### ISSUE TITLE
" Collect Mesh Contact Force

### ISSUE THREAD

**Issue by sarina1377** [projectchrono/chrono#410]
Hello,
I am trying to get the contact forces from particles to a defined mesh, but I can only get the sum of the forces using "Collectmeshcontactforce" in GPU.
Is there a way to get the individual contact forces from particles to the mesh for each particle, instead of just the sum of the contact forces on one mesh?

**Comment by rserban** [comment]
No, this is not possible. The individual contact forces are kept on the device and never transferred to the host.
